### PR TITLE
4.5 screenshot update

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | none |
| Versions      | 4.5/master |
| Edition       | all |

#### Checklist

- [x] Text renders correctly

### CODE REVIEW

**Review by DominikaK — APPROVED** [review]

File: docs/content_management/img/select_page_blocks.png
Comment (MagdalenaZuba) [inline docs/content_management/img/select_page_blocks.png:1]: Above **Select blocks** I can see a part of an arrow and some text that was not removed

File: docs/getting_started/img/left_menu_tree.png
Comment (MagdalenaZuba) [inline docs/getting_started/img/left_menu_tree.png:1]: This is the same picture

File: docs/getting_started/img/top_bar.png
Comment (MagdalenaZuba) [inline docs/getting_started/img/top_bar.png:1]: I cannot spot a difference

**Review by MagdalenaZuba — CHANGES_REQUESTED** [review]

File: docs/getting_started/img/left_menu_tree.png
Comment (dabrt) [reply docs/getting_started/img/left_menu_tree.png:1]: It is not, the "State" pulldown has changed.

Comment (dabrt) [reply docs/getting_started/img/left_menu_tree.png:1]: It is not, the colors on the left hand menu have changed ever so slightly for whatever reason between releases.

File: docs/getting_started/img/top_bar.png
Comment (dabrt) [reply docs/getting_started/img/top_bar.png:1]: Two changes - previous version showed a little background and the colors have changed very slightly.

File: docs/getting_started/img/left_menu_tree.png
Comment (MagdalenaZuba) [reply docs/getting_started/img/left_menu_tree.png:1]: I think it may be the difference in the monitor settings.

**Review by MagdalenaZuba — APPROVED** [review]

**Pull request merged**